### ISSUE TITLE
dev-python/gpiozero is missing `colorzero` dependency

### ISSUE THREAD

**Issue by denilsonsa** [onkelbeh/HomeAssistantRepository#22]
## How to reproduce:

I have Gentoo arm64 running on Raspberry Pi 4 B (installed using [sakaki-/gentoo-on-rpi-64bit](https://github.com/sakaki-/gentoo-on-rpi-64bit)).

1. Add `HomeAssistantRepository` to your portage overlays.
2. `emerge -av gpiozero`

        [ebuild  N     ] dev-python/gpiozero-1.5.1::HomeAssistantRepository  USE="-test" PYTHON_TARGETS="python3_6 python3_7 -python3_8" 140 KiB

3. Check if everything was installed: `equery files gpiozero`

        … Many files, but mostly: …
        /usr/bin/pinout -> ../lib/python-exec/python-exec2
        /usr/lib/python3.7/site-packages/gpiozero-1.5.1-py3.7.egg-info…
        /usr/lib/python3.7/site-packages/gpiozero/…
        /usr/lib/python3.7/site-packages/gpiozerocli/__init__.py
        /usr/lib/python3.7/site-packages/gpiozerocli/pinout.py
        /usr/lib64/python3.6/site-packages/gpiozero-1.5.1-py3.6.egg-info/…
        /usr/lib64/python3.6/site-packages/gpiozero/…
        /usr/lib64/python3.6/site-packages/gpiozerocli/__init__.py
        /usr/lib64/python3.6/site-packages/gpiozerocli/pinout.py

4. Try running the gpiozerocli tool: `pinout`

## What happens:

```
$  pinout
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.7/pinout", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3251, in <module>
    @_call_aside
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'colorzero' distribution was not found and is required by gpiozero
```

## What should happen:

It should printout a text-based pinout on the terminal. Look at the screenshots:

* https://gpiozero.readthedocs.io/en/stable/cli_tools.html
* https://www.raspberrypi.org/documentation/usage/gpio/

## Alternative way to trigger the issue:

```
$ ipython
Python 3.7.8 (default, Jul  2 2020, 17:08:45)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.16.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import gpiozero
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-d4c0b0396b91> in <module>
----> 1 import gpiozero

/usr/lib/python3.7/site-packages/gpiozero/__init__.py in <module>
     95     MCP3304,
     96 )
---> 97 from .output_devices import (
     98     OutputDevice,
     99     DigitalOutputDevice,

/usr/lib/python3.7/site-packages/gpiozero/output_devices.py in <module>
     43 from threading import Lock
     44 from itertools import repeat, cycle, chain
---> 45 from colorzero import Color
     46 from collections import OrderedDict
     47 try:

ModuleNotFoundError: No module named 'colorzero'
```

## Why it happens:

Because [`gpiozero` depends](https://github.com/gpiozero/gpiozero/blob/v1.5.1/setup.py#L57-L59) on [`colorzero` python module](https://pypi.org/project/colorzero/).

## Solution:

Someone has to write an ebuild for `colorzero`, add it to the overlay, and add it as dependency of `gpiozero` (since version 1.5.0).

**Comment by onkelbeh** [comment]
Oh, missed [that one, they added it in 1.5.0](https://github.com/gpiozero/gpiozero/commit/55c57c497cb71a98e84ab025808a830bc0e7dd51#diff-2eeaed663bd0d25b7e608891384b7298)
Should be fixed with https://github.com/onkelbeh/HomeAssistantRepository/commit/71c1eb52adb1e98bfe6066aa49a4e06205a8a46a.

Did not test on arm, yet. But I have plans for that.
For now the modules are OK for me if they compile on amd64.
Thanks for that impressive bug report :-)

One question: Did you set up a cross compiler?

**Comment by denilsonsa** [comment]
No, I did not set up any cross-compiler! Since I don't have any other Gentoo system around here, the Raspberry Pi 4 is either [downloading binaries](https://github.com/sakaki-/gentoo-on-rpi-64bit#new-features-to-expedite-regular-system-updating), or compiling stuff by itself.

Also, I don't plan running homeassistant on gentoo (I have a dedicated Pi just for hass.io); I just found your overlay because I was curious about the `gpiozero` module.
